doc/tutorials/12-constant_pH/si_units.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import scipy.constants as const
+
+# physical constants
+# kg atomic mass unit
+amu = const.physical_constants['atomic mass constant'][0]
```

Comment:
if this file should be part of espresso in the future, we should not pollute the namespace with such global variables that are just aliases.

Reply:
fixed